From 469f71cb69bbd2449303fe6811e19e63209d3e3b Mon Sep 17 00:00:00 2001
From: Dave Taht <dave.taht@bufferbloat.net>
Date: Sun, 27 Oct 2013 12:01:29 -0700
Subject: [PATCH 2/3] pie: set limit to 1000 packets

A 200 packet limit is rather small for reverse traffic at higher
rates.
---
 net/sched/sch_pie.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/sched/sch_pie.c b/net/sched/sch_pie.c
index 2689c87..53fee65 100644
--- a/net/sched/sch_pie.c
+++ b/net/sched/sch_pie.c
@@ -81,7 +81,7 @@ static void pie_params_init(struct pie_params *params)
 	params->alpha = 2;
 	params->beta = 20;
 	params->tupdate = PSCHED_NS2TICKS(30 * NSEC_PER_MSEC);	  /* 30 ms */
-	params->limit = 200;			 /* default of 200 packets */
+	params->limit = 1000;			 /* default of 200 packets */
 	params->target = PSCHED_NS2TICKS(20 * NSEC_PER_MSEC);	  /* 20 ms */
 	params->ecn = false;
 	params->bytemode = false;
-- 
1.7.9.5

